### PR TITLE
Jhony/add python 312 support

### LINKED ISSUE #926: [xqueue] Test Python 3.11 and 3.12

### PR DESCRIPTION
**Pull request by jfavellar90**
This PR aims to close https://github.com/openedx/xqueue/issues/926

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jfavellar90! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: .github/workflows/mysql8-migrations-check.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
         pip uninstall -y mysqlclient
         pip install --no-binary mysqlclient mysqlclient
         pip uninstall -y xmlsec
-        pip install --no-binary xmlsec xmlsec
```
Comment (feanil) [inline .github/workflows/mysql8-migrations-check.yml:56]: Add a comment for why this is being pinned.

**Review by feanil — APPROVED** [review]
Looks great and the tests look like they're running with the correct Python versions.  I have one request for a comment and once that's added I think this can merge.

File: .github/workflows/mysql8-migrations-check.yml (hunk 1 above)
Comment (jfavellar90) [reply .github/workflows/mysql8-migrations-check.yml:56]: @feanil Done

**Comment by jfavellar90** [comment]
@feanil ready to merge :)

**Comment by openedx-webhooks** [comment]
@jfavellar90 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/xqueue&entry.867055334=https://github.com/openedx/xqueue/pull/932&entry.1484655318=https://github.com/jfavellar90&entry.752974735=2024-04-29+04:57&entry.1917517419=2024-04-29+19:18&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**